ToolBox/IQFeedDownloader/IQFeedDataDownloader.cs

```diff
@@ -13,8 +12,8 @@ namespace QuantConnect.ToolBox.IQFeedDownloader
     /// </summary>
     public class IQFeedDataDownloader : IDataDownloader
     {
-        private readonly IQConnect _iqConnect;
-        private readonly HistoryPort _historyPort;
+        // private readonly HistoryPort _historyPort;
```

Comment:
nit - Commented code

Reply:
good catch

Reply:
well, since Jared already approved, can we merge and remove in the next PR. makes more sense..